frontend/scripts/setup/mac_setup.sh

```
@@ -0,0 +1,303 @@
+        mv "$CMDLINE_TOOLS_DIR/temp/cmdline-tools" "$CMDLINE_TOOLS_DIR/latest"
+        rm -rf "$CMDLINE_TOOLS_DIR/temp"
+        rm commandlinetools.zip
+    fi
+
+    # Set up environment variables
+    echo_info "Configuring Android SDK environment variables..."
+    if ! grep -q "Android/Sdk" ~/.zshrc; then
+        echo "# Android SDK Environment Variables" >> ~/.zshrc
+        echo "export ANDROID_HOME=$ANDROID_SDK_DIR" >> ~/.zshrc
+        echo "export PATH=\$ANDROID_HOME/cmdline-tools/latest/bin:\$ANDROID_HOME/platform-tools:\$ANDROID_HOME/emulator:\$PATH" >> ~/.zshrc
+        source ~/.zshrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)